web/react-web/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM node:latest
+
+COPY . /app
+WORKDIR /app
+RUN npm install
+RUN npm run build
+RUN mv build /
+WORKDIR /build
+RUN rm -rf /app
```

If I get what's going on, these might be better names:
```suggestion
COPY . /app-source
WORKDIR /app-source
RUN npm install
RUN npm run build
RUN mv build /app
WORKDIR /app
RUN rm -rf /app-source
```